deposit_contract/contracts/validator_registration.v.py

```diff
@@ -52,6 +52,10 @@ def get_deposit_root() -> bytes32:
         size /= 2
     return root
 
+@public
+@constant
+def get_deposit_count() -> bytes[8]:
```

Comment:
10000000000000000000 deposit binance for ETH purchase tdmllc0@gmail.com transfer to tdmllc1@outlook.com